bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -76,6 +81,9 @@
     private String appResourcesPath;
     private NavigationItem redirectTarget;
 
+    protected String[] clientLibCategoriesJsBody = new String[0];
```

Comment:
I'd write `clientlib` in lower case -> `clientlibCategoriesJsBody`.

Reply:
@jckautzmann - I kept the camelCase for consistency with the existing `getClientLibCategories` API function as well as the `clientLibCategories` property. I think you could argue for camelCase in `clientLibrary` -> `clientLib`.

Reply:
ok, sure, let's be consistent with the existing API naming patterns.